ansible/roles/stig/tasks/TOSS_04_010020.yml

```
@@ -23,3 +45,4 @@
     - CCI-000186
     - isso_documented
     - pki_authentication
+    - manual
```

```suggestion
    - manual
    - hpc_issue
```

This is one that's going to be tricky for us generally / has some concerns, so let's add that tag